politeiawww/invoices.go

```diff
@@ -1592,6 +1594,20 @@ type backendInvoiceStatusChange struct {
 	Timestamp      int64              `json:"timestamp"`      // Timestamp of the change
 }
 
+// backendInvoicePayment represents an invoice payment and is stored
+// in the metadata stream mdStreamInvoicePayment in politeiad.
+type backendInvoicePayment struct {
```

Comment:
Seems like we're storing a lot of unnecessary data here.  A few comments:

Token - already in the `UpdateVettedMetadata` request
Address - already in the invoice
TimeStarted - is this the same time as the invoice is approved?  If so, do we really need this?
Status - is the payment status needed?  Isn't this mdstream just being updated once the invoice has been paid?
AmountNeeded - this could be either `ExchangeRate` or `AmountNeeded`.  Would be a good idea to use whichever one is used throughout the code more in order to minimize conversions.